nomenclature/definitions/variable/energy/energy-final.yaml

```diff
@@ -501,6 +501,13 @@ Final Energy|Electricity|Other (excl. Heat, Cooling, Transport):
    sector: all
    energy use: all excl.heating, cooling, transportation               
    unit: GWh/yr
+   
+Final Energy|Electricity|Other (excl. Heat, Transport):
```

Comment:
The variable name "Other" is usually meant as "everything that doesn't have an own subcategory" - whereas you seem to mean "every category except". Maybe "Total (excl. Heat and Transport)" would be a better identifier?

Reply:
@danielhuppmann @tburandt yes this would indeed make sense